### PR TITLE
ft(application-form):application form for new applicants

### PR DESCRIPTION
**Pull request by ba-lambert**
#### How should be tested
```
checkout to ft-new-applications-form
run npm install
open apollo client-mutations-createnewapplication 
```
```
you can use these to test endpoint
{
  "areYouEmployed":true,
    "atlpUnPaid":true,
    "currentlyStudying":true,
    "district":"gasabo",
    "doYouHaveLaptop":true,
    "dob":"2023-08-27",
    "educationLevel":"level1",
    "email":"fox4@gmail.com",
    "firstName":"myname",
    "gender":"male",
    "highestLevelOfEducation":"university",
    "howDidYouFindThisApplication":"twitter",
    "lastName":"lastname",
    "nationality":"rwandan",
    "participateAndelaPrograms":"yes",
    "province":"kigali city",
    "sector":"nyamirambo",
    "whatWasYourDiscipline":"MPC"
}
```

### CODE REVIEW

**Review by dean-daryl — APPROVED** [review]

**Review by JENNY-257 — APPROVED** [review]

**Review by JENNY-257 — APPROVED** [review]

**Review by JosephNgabo — APPROVED** [review]

**Comment by JosephNgabo** [comment]
there is some conflicts in index.ts you can resolve them

File: credentials.json [hunk 1]
```diff
@@ -9,4 +9,4 @@
     "redirect_uris": ["http://localhost:3001/", "http://localhost"],
     "javascript_origins": ["http://localhost", "http://localhost:3000"]
   }
```
Comment (Anguandia) [inline credentials.json:8]: Wouldn't we rather keep the credential values in the environment?

File: src/models/newApplications.ts
Comment (Anguandia) [inline src/models/newApplications.ts:1]: How shall we validate some items as required, considering conditional form flow? For example, if the value for "currentlyStudying" is False, then Required for education level is False. Please let's get together as a focus group and brainstorm this

**Review by Anguandia — CHANGES_REQUESTED** [review]

**Comment by Anguandia** [comment]
@ba-lambert , should we close this PR? if the functionality is already implemented through another proxy, please confirm asap

**Pull request closed without merging**